### PR TITLE
[test] Split ci values files

### PR DESCRIPTION
**Pull request by aledegano**
/deploy #persist

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -44,9 +56,11 @@ User-Facing Changes
 - **UI**: Prevent losing work when editing datasets
   (`#2628 <https://github.com/SwissDataScienceCenter/renku-ui/issues/2628>`_).
 
+
 Internal Changes
 ~~~~~~~~~~~~~~~~
 
+<<<<<<< HEAD
```
Comment (rokroskar) [inline CHANGELOG.rst:63]: oops
```suggestion
```

File: helm-chart/values.yaml.changelog.md [hunk 2]
```diff
@@ -6,6 +6,47 @@ Please follow this convention when adding a new row
...
+      user: <string>
+      password: <string>
+      slaveReplicas: <int>
+  ```
+New
+  ```
+  postgresql:
+    auth:
+      username: <string>
+      database: <string>
```
Comment (rokroskar) [inline helm-chart/values.yaml.changelog.md:35]:
```suggestion
      database: <string>
      username: <string>
```

**Review by rokroskar — CHANGES_REQUESTED** [review]
This makes sense! Can you please add a PR description to motivate the change?

**Comment by aledegano** [comment]
> This makes sense! Can you please add a PR description to motivate the change?

Sure, I'm still testing this PR, I'll repackage it more sensibly when it'll work :)

**Comment by rokroskar** [comment]
OK sure - should it be a draft then?

**Pull request closed without merging**